docs/src/architecture/10_quality/testplans/flutter_cardano_integration.md

```diff
@@ -0,0 +1,128 @@
+<!--
+Based on: https://books.google.it/books?id=vHlTOVTKHeUC&hl=it&source=gbs_navlinks_s
+          https://testing.googleblog.com/2016/06/the-inquiry-method-for-test-planning.html
+          https://testing.googleblog.com/2011/09/10-minute-test-plan.html
+-->
+
+# [Flutter Cardano Integration](https://github.com/input-output-hk/catalyst-voices/issues/481)
```

Comment:
Can you link the actual issue with the testplan instead of the template one?

Reply:
done